src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -788,7 +788,7 @@ private PollingResult compareRemoteRevisionWithImpl(Job<?, ?> project, Launcher
             return BUILD_NOW;
         }
 
-        GitClient git = createClient(listener, environment, project, node, workingDirectory);
+        GitClient git = createClient(listener, environment, project, project.getLastBuild(), node, workingDirectory);
```

Comment:
Same question as above.  This is in a method that initialized a `lastBuild` value on entry to the method.  It seems quite safe to call `project.lastBuild()` to get the value again, but is that intentional for some reason that I missed?